src/puya/awst/to_code_visitor.py

```diff
@@ -472,6 +472,13 @@ def visit_update_inner_transaction(self, expr: nodes.UpdateInnerTransaction) ->
             fields.append(f"{field.immediate}={value.accept(self)}")
         return f"update_inner_transaction({expr.itxn.accept(self)},{', '.join(fields)})"
 
+    @typing.override
+    def visit_stage_inner_transactions(self, node: nodes.StageInnerTransactions) -> str:
+        start_new_group = node.start_new_group.accept(self)
+        itxns = f'{", ".join(itxn.accept(self) for itxn in node.itxns)}'
+
+        return f"stage(itxns=[{itxns}], start_new_group={start_new_group})"
```

Comment:
Not that it matters too much, but maybe we should call this `stage_itxns` instead of having the `itxns=` bit?
Just because otherwise it might look like a subroutine name. I mean, either way, this output here is mostly for debugging the front end so it doesn't matter too much.